pkg/common/security/acl.go

```diff
@@ -30,7 +30,10 @@ const (
     Space     = " "
 )
 
-var UserNameRegExp = regexp.MustCompile("^[a-z_]([a-z0-9_-]{0,31}|[a-z0-9_-]{0,30}\\$)$")
+// User and group regexp, must allow at least what we allow in the config checks
+// See configs.UserNameRegExp and configs.GroupRegExp
+var UserNameRegExp = regexp.MustCompile("^[_a-zA-Z][a-zA-Z0-9_.@-]*[$]?$")
+var GroupRegExp = regexp.MustCompile("^[_a-zA-Z][a-zA-Z0-9_-]*$")
```

Comment:
I am wondering why `UserNameRegExp` and `GroupRegExp ` instead of `userNameRegExp`/`groupRegExp`, since this regex is only used in this file.

Reply:
They should have been private.
Fixed both of them